### PR TITLE
no keepupHandler if typing in medium editor

### PR DESCRIPTION
**Pull request by pchatterton**

### CODE REVIEW

File: utils/keybinding.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ exports.hasKeybinding = function (key) {
 
 function keyupHandler (matchChar, callback, evt) {
   // we don't want to fire events when somebody is typing in an input, esc is okay to fire
-  if ((evt.target.tagName !== 'INPUT' && evt.target.tagName !== 'TEXTAREA') || evt.keyCode === KEY_NAMES.esc) {
+  if ((evt.target.tagName !== 'INPUT' && evt.target.tagName !== 'TEXTAREA' && !evt.target.hasAttribute('data-medium-editor-element')) || evt.keyCode === KEY_NAMES.esc) {
```
Comment (jakerichan) [inline utils/keybinding.js]: don't reference the data-medium-editor, that could change and not everybody uses that component

Comment (jakerichan) [inline utils/keybinding.js]: check for content-editable

Comment (pchatterton) [inline utils/keybinding.js]: :thumbsup:

**Comment by jakerichan** [comment]
:+1:

**Pull request merged**